gui/re.chiaki.Chiaki.appdata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 Florian Märkl <chiaki@metallic.software> -->
+<component type="desktop">
```

Comment:
Nitpick: `desktop-application` instead of `desktop` (but either will work fine)

Reply:
I'd prefer not to introduce more changes in this PR.
It was desktop in previous version too. Let's keep it this way :)